pkg/controllers/networkinfo/networkinfo_utils.go

```diff
@@ -83,12 +84,12 @@ func setVPCNetworkConfigurationStatusWithLBS(ctx *context.Context, client client
 			return
 		}
 	}
-	// else append the new VPCInfo
-	if nc.Status.VPCs == nil {
-		nc.Status.VPCs = []v1alpha1.VPCInfo{}
-	}
 	nc.Status.VPCs = append(nc.Status.VPCs, *createdVPCInfo)
-	client.Status().Update(*ctx, nc)
+	err = client.Status().Update(*ctx, nc)
+	if err != nil {
+		log.Error(err, "Update VPCNetworkConfiguration status failed", "ncName", ncName, "vpcName", vpcName, "nc.Status.VPCs", nc.Status.VPCs)
```

Comment:
line 90 and 92 maybe all printed in the log if err != nil, a little confusing maybe add else for line 92?

Reply:
Good catch, and it was fixed in https://github.com/vmware-tanzu/nsx-operator/pull/681